topics/groovy/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: groovy-lang
```

Comment:
`groovy-lang` isn't currently used on any public repositories, see https://github.com/topics/groovy-lang. Could you remove this line?